modules/querier/querier.go

```diff
@@ -171,34 +171,35 @@ func (q *Querier) FindTraceByID(ctx context.Context, req *tempopb.TraceByIDReque
 				if spanCount > 0 {
 					spanCountTotal = spanCount
 				}
-				span.LogFields(ot_log.String("msg", "combined trace protos from ingesters"))
 			}
 		}
-
-		span.LogFields(ot_log.String("msg", "done searching ingesters"), ot_log.Bool("found", completeTrace != nil))
-	}
-
-	span.LogFields(ot_log.String("msg", "searching store"))
-	partialTraces, err := q.store.Find(opentracing.ContextWithSpan(ctx, span), userID, req.TraceID, req.BlockStart, req.BlockEnd)
-	if err != nil {
-		return nil, errors.Wrap(err, "error querying store in Querier.FindTraceByID")
-	}
-
-	span.LogFields(ot_log.String("msg", "done searching store"))
-	// combine partialTraces with completeTrace
-	for _, partialTrace := range partialTraces {
-		storeTrace := &tempopb.Trace{}
-		err = proto.Unmarshal(partialTrace, storeTrace)
+		span.LogFields(ot_log.String("msg", "done searching ingesters"),
```

Comment:
leave this logic the way it was.  even though we no longer are creating queries with both blocks and ingester the querier should still respond correctly if it gets that query.

Reply:
There's no way to differentiate between a query from the Query Frontend that has `queryIngesters=true` and another fired directly to the querier with the flag.. Are you saying we need to perform different operations for each of these?

I would prefer we don't optimise for the debugging use-case.